port/ztpf/omrvmem.c

```diff
@@ -1480,6 +1481,7 @@ getMemoryInRangeForDefaultPages(struct OMRPortLibrary *portLibrary,
 				}
 			}
 		}
+#endif
```

Comment:
https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#comment-conditional-compilation-directives-if-ifdef-ifndef-else-elif-endif
```suggestion
#endif /* OMRZTPF */
```

Reply:
Done